WeakAuras/Prototypes.lua

```diff
@@ -327,7 +327,7 @@ WeakAuras.function_strings = {
   ]],
   count_fraction = [[
     return function(count, max)
-      local fraction = count/max
+      local fraction = count/(max ~= 0 and max or math.huge) -- because PTR always complains about div/0
```

Comment:
I don't know if infinity is the correct option, or if we should go with 1. Technically I think no matter what we do, this would make unintuitive results whenever blizzard is giving us silly data.

Reply:
Having thought about it, imho the best was to handle this is that the count_fraction method returns false for max == 0.
so basically.

Reply:
I did that now.